scalableminds/webknossos
# Add layer + mag dir to wkw uploads if missing

When a user uploads wkw-converted data directly, that is not in a layer directory or not even in a mag directory, the back-end now automatically adds those to the path.

A risk to this method is that layers called `1`, `1-1-1` or `z0` etc would be mistaken for mags/bucket paths and this would trigger wrongly. But I’ll assume this will rarely be the case.

### URL of deployed dev instance (used for testing):
- https://uploadwkwdir.webknossos.xyz

### Steps to test:
- upload only parts of the dataset
- should be moved (console logging), and viewable after import

### Issues:
- fixes #5219 

------
